src/beautifiers/rubocop.coffee

```diff
@@ -16,53 +18,50 @@ module.exports = class Rubocop extends Beautifier
       rubocop_path: true
   }
 
-  beautify: (text, language, options) ->
+  beautify: (text, language, options, context) ->
+    fullPath = context.filePath
+    [projectPath, relativePath] = atom.project.relativizePath(fullPath)
+
+    # Find the rubocop path
+    rubocopPath = "rubocop"
     @Promise.all([
       @which(options.rubocop_path) if options.rubocop_path
       @which('rubocop')
-    ]).then((paths) =>
+    ])
+    .then((paths) =>
       @debug('rubocop paths', paths)
-      _ = require 'lodash'
-      path = require 'path'
       # Get first valid, absolute path
-      rubocopPath = _.find(paths, (p) -> p and path.isAbsolute(p) )
+      rubocopPath = paths.find((p) -> p and path.isAbsolute(p))
       @verbose('rubocopPath', rubocopPath)
       @debug('rubocopPath', rubocopPath, paths)
+    )
```

Comment:
I think this should be at the end of the file. The code after this line does not appear to wait for the result of this promise.